packages/mds-audit/api.ts

```diff
@@ -665,6 +696,59 @@ function api(app: express.Express): express.Express {
     }
   })
 
+  /**
+   * attach media to an audit, uploaded as multipart/form-data
+   */
+  app.post(pathsFor('/trips/:audit_trip_id/attach/:mimetype'), multipartFormUpload, async (req, res) => {
+    const { audit, audit_trip_id } = res.locals
+    if (!audit) {
+      return res.status(404).send({ error: new NotFoundError('audit not found', { audit_trip_id }) })
+    }
+    try {
+      validateFile(req.file)
+    } catch (err) {
+      if (err instanceof ValidationError) {
+        return res.status(400).send({ error: err })
+      }
+      if (err instanceof UnsupportedTypeError) {
+        return res.status(415).send({ error: err })
+      }
+    }
+    const attachment: Attachment | null = await writeAttachment(req.file, audit_trip_id)
+    if (attachment) {
+      res.status(200).send({
+        ...attachmentSummary(attachment),
+        audit_trip_id
+      })
+    } else {
+      await log.error('post attachment fail')
+      res.status(500).send({
+        error: 'server_error',
+        error_description: 'an internal server error has occurred and been logged'
+      })
+    }
+  })
+
+  /**
+   * delete media from an audit
+   */
+  app.delete(pathsFor('/trips/:audit_trip_id/attachment/:attachment_id'), async (req, res) => {
+    const { audit_trip_id, attachment_id } = req.params
+    try {
+      await deleteAuditAttachment(audit_trip_id, attachment_id)
+      res.status(200).send({})
+    } catch (err) {
+      await log.error('delete attachment error', err)
+      if (err instanceof NotFoundError) {
+        return res.status(404).send({ error: err })
+      }
+      res.status(500).send({
+        error: 'server_error',
+        error_description: 'an internal server error has occurred and been logged'
+      })
```

Comment:
We should attempt to make our error bodies consistent. Most of the time we return `{ error: err }`